go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/AlecAivazis/survey/v2 v2.3.6
 	github.com/acorn-io/aml v0.0.0-20230418220934-5635aac007d1
 	github.com/acorn-io/baaah v0.0.0-20230410050548-f08516abc5c3
-	github.com/acorn-io/mink v0.0.0-20230408054940-c2216a3c8f8d
+	github.com/acorn-io/mink v0.0.0-20230420172810-2d840529cb63
```

Comment:
This looks like it's a pseudoversion from last Thursday -- I just pulled `@main` and got `v0.0.0-20230424190003-9a32355ec823`

Reply:
Thanks for catching that. Apparently I ran `go get` to update it a little too quickly after I merged my mink PR, lol. I was wondering why the timestamp looked off before. I pushed a commit to fix this.